R/parameters.R

```diff
@@ -1,11 +1,28 @@
 #' @title Default parameters for DAEDALUS
+#' @description Useful defaults for the DAEDALUS model for use during
+#' development.
 #' @param ... User-specified values to replace default parameter values.
 #' @export
 #' @examples
 #' default_parameters(beta = 1.5 / 7.0)
 default_parameters <- function(...) {
   user_params <- list(...)
 
+  contact_matrix <- matrix(1.0, N_AGE_GROUPS, N_AGE_GROUPS)
+
+  # workplace contacts within sectors
+  cmw <- rep(10.0, N_ECON_SECTORS - 1L)
+
+  # NOTE: default assumption is uniformly distributed contacts
+  cmcw <- matrix(
+    rep(
+      rep(1.0, N_AGE_GROUPS),
+      N_ECON_SECTORS
+    ),
+    nrow = N_ECON_SECTORS,
+    ncol = N_AGE_GROUPS
+  )
```

Comment:
```suggestion
  cmcw <- matrix(1.0, N_ECON_SECTORS, N_AGE_GROUPS)
```

sorry, should have spotted this before